test/test.js

```diff
@@ -61,15 +61,20 @@ describe('svg2ttf', function () {
 
 
   describe('os/2 table', function () {
+    let parsed = opentype.parse(svg2ttf(fixture).buffer.buffer);
+    let os2 = parsed.tables.os2;
```

Comment:
FYI, avoid shared variables it tests without need (very complex init). When test is self-sufficient - less chances to destroy it from outer code. That's specific recommendation for tests - "avoid shared state"

Reply:
Do you mean to redefine the `parsed` and `os2` variables once in each  `it()` function?